crates/crabe_decision/src/strategy/testing/test_vision_moveto.rs

```diff
@@ -0,0 +1,79 @@
+use std::ops::Div;
+use nalgebra::{distance, Point2};
+use crabe_framework::data::output::Command;
+use crabe_framework::data::tool::ToolData;
+use crabe_framework::data::world::World;
+use crate::action::ActionWrapper;
+use crate::action::move_to::MoveTo;
+use crate::strategy::Strategy;
+
+const DIST_TARGET_REACHED: f64 = 0.1;
+
+#[derive(Debug)]
+enum TestVisionMoveToStatus {
+    Placement,
+    MovingForward,
+    MovingBackwards,
+}
+
+pub struct TestVisionMoveTo {
+    ids: Vec<u8>,
+    status: TestVisionMoveToStatus,
+    positive_half: bool,
+}
+
+impl TestVisionMoveTo {
+    pub fn new(ids: Vec<u8>, positive_half: bool) -> Self {
+        Self {
+            ids,
+            status: TestVisionMoveToStatus::Placement,
+            positive_half,
+        }
+    }
+
+
+}
+
+impl Strategy for TestVisionMoveTo {
+    fn name(&self) -> &'static str {
+        "TestVisionMoveTo"
+    }
+
+    fn step(&mut self, world: &World, _: &mut ToolData, action_wrapper: &mut ActionWrapper) -> bool {
+        // WARNING : Not clearing the action_wrapper leads to stuttering
+        action_wrapper.clear_all();
```

Comment:
There's an issue at hand; running two tests in parallel seems to be unfeasible with this line.
Could you please investigate the cause of the robot's stuttering before merging ?

Reply:
But don't we only run a single strategy at a time ? I see your point though, this isn't ideal because if we could launch two strategies in parallel, the second strategy would literally override the first one.

Reply:
Why should an override be necessary if a different strategy is intended for another robot than this? 
Multiple strategies can run concurrently without broken.

As of now, we haven't implemented any measures to prevent the scenario where two strategies claim the same robot.

Reply:
I meant this is currently the issue lol
Anyways how do we fix this ? Should we open a new issue for now and take care of this later ?

Reply:
Each action possesses a status, which facilitates the resolution of the initial issue.

In the scenario where two strategies vie for the same robot, the resolution becomes somewhat challenging. However, such scenarios fall outside the intended purview of these strategies.